drivers/eeprom/i2c_xx24xx.c

```diff
@@ -253,7 +253,7 @@ static int ee24xx_waitwritecomplete(FAR struct ee24xx_dev_s *eedev,
 {
   struct i2c_msg_s msgs[1];
   int ret;
-  int retries = 100;
+  int retries = 500;
```

Comment:
Reading the note above.

>Note: We should sleep a bit between retries, the write time is around 5 ms,
> but the bus is slow, so, a few retries at most will happen.	 

makes me think the proper solution should use time (or add a up_udelay()) not a count.

Perhaps something along the lines of https://github.com/PX4/NuttX/blob/px4_firmware_nuttx-8.2/arch/arm/src/stm32/stm32f40xxx_i2c.c#L723-L740 

What to you think?